wsl-pro-service/debian/rules

```diff
@@ -14,6 +14,11 @@ export GOTOOLCHAIN := local
 	dh $@ --builddirectory=_build --with=apport
 
 execute_after_dh_auto_clean:
+	# Overriding version file
+	# We must do this during the source package build because we still have access to git.
+	# We can't do this during the binary package, so we cannot use the usual -ldflags.
```

Comment:
Be careful that debian/rules clean is called during package binary build too. Maybe we should have a `if source build` do that, and put everything under it (vendoring and so on).

I suggest that we only trigger this in one way.

Reply:
> Be careful that debian/rules clean is called during package binary build too

Yeah, that if is there to avoid calling it during binary build. I agree though, a single isSource signal would be best. You think the vendor cek is good enough?

Reply:
I’m not a fan of either approach, I’m looking if we can have a better decision factor.